src/_pytest/nodes.py

```diff
@@ -128,8 +128,8 @@ def __init__(
         #: keywords/markers collected from all scopes
         self.keywords = NodeKeywords(self)
 
-        #: the marker objects belonging to this node
-        self.own_markers = []  # type: List[Mark]
+        #: The (manually added) marks belonging to this node (start, end).
```

Comment:
What does "start", "end" mean? (=> would be good to explain in the comment)

Reply:
@bluetech 
It keeps two list for prepended/appended marks - with the ones from the object inbetween.  This might not be needed really, but is needed for the current behavior (fixing it for changed `obj`s etc though).
Having two separate lists might be good, but also not really needed maybe (given that the single list can be typed "enough" already).
Any suggestions for a better/clearer comment?

Reply:
> Having two separate lists might be good, but also not really needed maybe

Yes, looking a bit more I can see either way.

> Any suggestions for a better/clearer comment?

First I think the name `_own_markers` is a little misleading now: the established meaning of `own_markers` (which can't be changed due to backward compat) includes the manually-added marks (`add_marker()`) but also the marks directly applied to the object (`@pytest.mark.foo`). But after this change, `_own_markers` only contains the `add_marker()` ones.

So maybe `_manually_added_markers`?

As regards to the comment, maybe something like this?

```
Marks that are manually added to this node with `add_marker()`.
These marks are included in the node's `own_markers`. Marks in the first item are prepended,
and marks in the second item are appended.
```